DQM/EcalMonitorClient/src/OccupancyClient.cc

```diff
@@ -167,7 +178,41 @@ namespace ecaldqm
         meQualitySummary.setBinContent(id, meQualitySummary.maskMatches(id, mask, statusManager_) ? kMBad : kBad);
       }
     }
-  }
+
+    // Quality check: set entire FED to BAD if its occupancy begins to vanish
+    // Fill FED statistics from (filtered) RecHit Occupancy 
+    std::vector<float> FEDStatsEB;
+    std::vector<float> FEDStatsEE;
```

Comment:
@mbandrews 
since it looks like FEDStatsE* are not used except to compute a mean and rms, I would suggest something like this pseudocode instead. It avoids both the vector construction and the TMath dependency. Could you have a look at changing it?

float meanFEDEB(0), meanFEDEE(0), rmsFEDEB(0), rmsFEDEE(0);
unsigned int cFEDEB(0), cFEDEE(0);

for ( unsigned iDCC(0); iDCC<nDCC; iDCC++)
   if ( idCC >=kEBmLow && iDCC <= kEBpHigh) {
       meanFEDEB+=Nrhentries[iDCC];
       rmsFEDEB+=Nrhentries[iDCC]*Nrhentries[iDCC];
       cFEDEB++;
}
else{
       meanFEDEE+=Nrhentries[iDCC];
       rmsFEDEE+=Nrhentries[iDCC]*Nrhentries[iDCC];
       cFEDEE++;
}

meanFEDEB/=float(cFEDEB);
meanFEDEE/=float(cFEDEE);
rmsFEDEB= sqrt(rmsFEDEB/float(cFEDEB));
rmsFEDEE= sqrt(rmsFEDEE/float(cFEDEE));